build_in_docker/file_cavity_detection.py

```diff
@@ -92,26 +105,29 @@ def file_cavity_detection(file: Path, output_dir: Path, timeout: int) -> str:
         if return_str == "":
             command = [
                 "python3",
-                str(SCRIPTDIR / "../polytracker/dumptdag.py"),
+                # str(SCRIPTDIR / "../polytracker/dumptdag.py"),
+                str(SCRIPTDIR / "cavities.py"),
```

Comment:
I guess the `dumptdag.py` script is about to be retired :)

Reply:
Better leave it for a while longer. The polytracker one is still pretty slow :eyes: